### PR TITLE
WIP async migration for adding event to events table sort key in cloud

### PR DESCRIPTION
**Pull request by macobo**
Solves the other half of https://github.com/PostHog/posthog/issues/5684 for cloud only.

I promised to take a first stab at this - @yakkomajuri will pick it up from here.

Added comments everywhere I'm not super sure on.

### CODE REVIEW

File: posthog/async_migrations/migrations/0002_events_sort_key_include_event.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+            sql=f"""
+            INSERT INTO {TEMPORARY_TABLE_NAME}
+            SELECT *
+            FROM {EVENTS_TABLE}
+            WHERE timestamp < toDateTime64(toYYYYMM(now()) - 1, 6)
+            AND timestamp >= (SELECT max(timestamp) FROM {TEMPORARY_TABLE_NAME})""",
+            rollback=f"TRUNCATE TABLE IF EXISTS {TEMPORARY_TABLE_NAME} ON CLUSTER {CLICKHOUSE_CLUSTER}",
+            resumable=True,
+            # Karl: Needs to run on all our ch nodes
+            run_on_all_nodes=True,
```
Comment (yakkomajuri) [inline posthog/async_migrations/migrations/0002_events_sort_key_include_event.py:63]: note to self: we still need to build this

Comment (yakkomajuri) [reply posthog/async_migrations/migrations/0002_events_sort_key_include_event.py:63]: would also be useful for the DLQ table migration

**Comment by macobo** [comment]
https://github.com/PostHog/posthog/pull/7778 fixes this

**Comment by macobo** [comment]
Closing - this was solved by the other async migration.

**Pull request closed without merging**